oci/private/push.sh.tpl

```diff
@@ -6,12 +6,17 @@ readonly YQ="{{yq_path}}"
 readonly IMAGE_DIR="{{image_dir}}"
 readonly TAGS_FILE="{{tags}}"
 readonly FIXED_ARGS=({{fixed_args}})
+readonly REPOSITORY_FILE="{{repository_file}}"
+
+REPOSITORY=""
+if [ -f $REPOSITORY_FILE ] ; then
+  REPOSITORY=$(tr -d '\n' < "$REPOSITORY_FILE")
+fi
 
 # set $@ to be FIXED_ARGS+$@
 ALL_ARGS=(${FIXED_ARGS[@]} $@)
 set -- ${ALL_ARGS[@]}
 
-REPOSITORY="{{repository}}"
```

Comment:
Looks like we no longer interpolate the `repository` variable anywhere in this template? That doesn't seem correct.

Reply:
When I looked at the file it seemed like the repository variable was never stamped anyway, I assumed it was an oversight - it is currently stamped into the `fixed_args` variable instead

Reply:
I see, there was no `repository` variable being interpolated to begin with.